format_and_overlap_determination.md

```diff
@@ -0,0 +1,118 @@
+
+**n_enum**
+: A normalized enumeration. Description fields are processed for enumeration and chronology. Only enumeration ends up being used. 
+
+
+## Item Format ##
+
+**Serial**
+: Items with a **Bib Record ID** found on the **Print Serials List**, or in a **Large Cluster** 
+
+**Multi-part Monograph**
+: Is not a Serial. Shares a **Bib Record** with an item that has a non-empty `n_enum`. [^1]
+
+**Single-part Monograph**
+: Is not a Serial or Multi-part
+
+**NB**: A cluster may contain items of multiple formats.  
+**NB2**: Format as reported by members is irrelevant.
+
+## Cluster Format ##
+
+**Multi-part Monograph**
+: Any of the items in the cluster are MPMs.
+
+**SER / SPM**
+: Is not MPM. One or more of the items in the cluster are SPM _and_ one or more of the items in the cluster are SER.
+
+**SER**
+: Is not MPM or SER/SPM. Any of the items in the cluster are SER (and by implication ALL are SER).
+
+**SPM**
+: Is not MPM, SER/SPM, or SER.
+
+## Cost Allocation ##
+
+**Target Cost**
+: Set by the Board and Executive Director. Portion of the budget to be paid for by item allocations.
+
+**Cost Per Volume**
+: Target Cost / Total Number of HT Items
+
+**Member Weight**
+: A number assigned to a member by the ED that determines their share of Public Domain costs. 
+
+**Total Weight**
+: The sum of member weights found in the `ht_billing_members` table of the holdings database. **Which process/system has ownership of this table?**
+
+**Public Domain Cost**
+: Cost Per Volume * Number of Public Domain Volumes (see Public Domain vs In-Copyright)
+
+**Public Domain Cost for Member**
+: Total Public Domain Cost / Total Weight * Member Weight
+
+**Extra Per Member**
+: The IC Costs assigned to Hathitrust / Number of members found in the `ht_billing_members` table that aren't Hathitrust. Hathitrust has IC costs due to being the billing entity for some items.
+
+**HScore**
+: A share of an HT Item.
+
+**Total HScore (for Member)**
+: Sum of HScores for Member for each of SPM, MPM and SER
+
+**Total IC Cost for Member**
+: Total HScore for Member * Cost Per Volume
+
+**Total Cost for Member**
+: Total IC Cost for Member + Total PD Cost for Member + Extra Per Member
+
+
+### Public Domain vs In-Copyright ###
+For cost allocation purposes, items with access "allow" in the Hathifiles are considered "public domain" and access "deny" are considered "in copyright". Rationale for the "allow" and "deny" found in the Hathifiles is underdefined. 
+
+## Item Overlap ##
+Determining who holds a particular IC item.
+
+For items in **clusters** that are not MPM, all organizations with a holding in the cluster and the billing entity for the item are allocated a share. 
+
+For items in **clusters** that are MPM, the process is more complicated. Organizations with holdings:
```

Comment:
A topic of conversation for our weekly meeting.

Reply:
What about:

For items in *clusters* that are not MPM (i.e. none of the items in the cluster have the item format multi-part monograph), organizations are considered to hold the item and are allocated a share for it if:
  * They are the billing entity for the item  (i.e. the depositor of an item is always assumed to hold it)
  * They submitted a holding that is in the cluster
 
 For items in clusters that are MPM, the process is more complicated. Organizations are considered to hold the item and are allocated a share if:

* They are the billing entity for the item (i.e. the depositor of an item is always assumed to hold it)
* They have a holding in the cluster with an n_enum matching the n_enum of an item.
* They have a holding in the cluster that has an empty n_enum. The same is not true of items with empty n_enums: holdings with empty n_enums match all items in an MPM cluster; but items with empty n_enums match only holdings with n_enums.
* They have holdings in the cluster, but none of the reported n_enums match any of the Item n_enums: that is, if no holdings n_enums match, then this is assumed to be a data problem, and the organization is considered to hold all the items in the cluster. [^2]

@nfulkers  Does that address any of the confusion? I think it is important this section be as clear as possible. I don't know whether my option is any better. It may be helpful for you to try stating the behavior in your words as you understand it, and then seeing if there's any spots we need to correct or clarify.

Reply:
I edited the above to replace 'organization' with 'institution'.

Reply:
Might need some more introductory text to make it clear that we are not actually matching holdings -- we are figuring out who holds an item, which includes (but is not limited to) matching holdings to HT items

Reply:
@nfulkers believes these revisions address the confusion